docs/deployment-guide.md

```diff
@@ -450,6 +450,36 @@ Once ACS has initialized access the system using the following URLs with a brows
 * Repository: `http://<nginx-host-ip>/alfresco`
 * API Explorer: `http://<nginx-host-ip>/api-explorer`
 
+### ACS cluster
+
+Due to load or high availability needs, you might want to deploy a cluster of several repository nodes. This can be achieved rather simply by:
+
+* Giving the playbook the location of the shared storage used for the ACS contentstore (See [Shared storage documentation](shared-contentstore.md) for details).
+* Specifying several hosts within the repository hosts group
+
+For example:
+
+````yaml
+...
+    repository:
+      vars:
+        cs_storage:
+          type: nfs
+          device: nas.infra.local:/exports/contentstore
+          options: _netdev,noatime,nodiratim
+      hosts:
+        ecm1.infra.local:
+        ecm2.infra.local:
+        ingester.infra.local:
+          cluster_keepoff: true
+...
+```
+
+In some circumstances, you may want to have a repo node that's dedicated to a scheduled task (such as ingesting massive amount of documents). Depending on the nature of the task and the requirements of your organisation, it may be preferable to not make this node part of the ACS cluster.
+In that case, you can add the `cluster_keepoff` variable to one of the `repository` group nodes'. It will provision the node with teh repository and share services but make sure it not taking part in neither the share, nor the repository cluster realm.
```

Comment:
```suggestion
In that case, you can add the `cluster_keepoff` variable to one of the `repository` group nodes'. It will provision the node with the repository and share services but make sure it not taking part in neither the share, nor the repository cluster realm.
```